app-text/nfoview/nfoview-1.28.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS=no
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1 xdg-utils
+
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/otsaloma/nfoview.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/otsaloma/nfoview/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="Simple viewer for NFO files, which are ASCII art in the CP437 codepage"
+HOMEPAGE="https://otsaloma.io/nfoview/"
+
+LICENSE="GPL-3+"
+SLOT="0"
+IUSE=""
+
+BDEPEND="${PYTHON_DEPS}
+	sys-devel/gettext"
+DEPEND="dev-python/pygobject:3[${PYTHON_USEDEP}]"
+RDEPEND="${DEPEND}
+	media-fonts/cascadia-code"
```

Comment:
Is this **your** preference, or really required by upstream? I couldn't find an answer while fast-glancing their GH page.

(Doesn't upstream prefer cantarell?)

Reply:
they recently switched the default font, since Terminus (the previous default) was a bitmap font and bitmap fonts are no longer supported by recent Pango releases.

see https://github.com/otsaloma/nfoview/commit/1eccb8e92107f988a03fa17f3f8e4887f96a7948

Reply:
one more word: Not every monospace font supports ANSI art equally well. Thus, installing the default here (and not fall back to the system default monospace font) should be preferred.